geoengine/workflow.py

```diff
@@ -95,6 +95,18 @@ def get_result_descriptor(self) -> ResultDescriptor:
 
         return self.__result_descriptor
 
+    def workflow_definition(self) -> Dict[str, str]:
```

Comment:
is the return type correct? 
 - key may be a string but the value can also be another operator etc.
 - I think we also have operators that have bool/number params

Also:
maybe we should have functions to handle json-input instead of Python Dicts

Reply:
28ef1e9 

JSON objects are mapped to dicts in Python.